### DISCUSSION TITLE
2017/08/26/testing-and-debugging-library-code-from-linqpad/

### DISCUSSION THREAD

**Discussion by giscus** [thomaslevesque/blog#114] [Blog comments]
# 2017/08/26/testing-and-debugging-library-code-from-linqpad/

https://thomaslevesque.com/2017/08/26/testing-and-debugging-library-code-from-linqpad/

**Comment by sixdiamants** [comment]
Pourquoi diable ignorais-je cette fonctionnalité jusqu'alors ?! 
D'autant plus utile pour vérifier l'état des objets COM que LINQpad n'expose pas. 
Un grand merci, Bob
